### ISSUE TITLE
A Meta-Learning Approach to the Optimal Power Flow Problem Under Topology Reconfigurations. (arXiv:2012.11524v1 [cs.LG])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#2203]
A Meta-Learning Approach to the Optimal Power Flow Problem Under Topology Reconfigurations. (arXiv:2012.11524v1 [cs.LG])<br>
https://ift.tt/3rivinI<br>
<p>Recently, there has been a surge of interest in adopting deep neural networks (DNNs) for solving the optimal power flow (OPF) problem in power systems. Computing optimal generation dispatch decisions using a trained DNN takes significantly less time when compared to using conventional optimization solvers. However, a major drawback of existing work is that the machine learning models are trained for a specific system topology. Hence, the DNN predictions are only useful as long as the system topology remains unchanged. Changes to the system topology (initiated by the system operator) would require retraining the DNN, which incurs significant training overhead and requires an extensive amount of training data (corresponding to the new system topology). To overcome this drawback, we propose a DNN-based OPF predictor that is trained using a meta-learning (MTL) approach. The key idea behind this approach is to find a common initialization vector that enables fast training for any system topology. The developed OPF-predictor is validated through simulations using benchmark IEEE bus systems. The results show that the MTL approach achieves significant training speeds-ups and requires only a few gradient steps with a few data samples to achieve high OPF prediction accuracy.</p>
<br>
<br>
via eess.SY updates on arXiv.org http://arxiv.org/<br>
